libpod/container_api.go

```diff
@@ -794,21 +794,25 @@ type ContainerCheckpointOptions struct {
 	// container no PID 1 will be in the namespace and that is not
 	// possible.
 	Pod string
+	// PrintStats tells the API to fill out the statistics about
+	// how much time each component in the stack requires to
+	// checkpoint a container.
+	PrintStats bool
 }
 
 // Checkpoint checkpoints a container
```

Comment:
Can you add a comment that we only return the Statistics struct if PrintStats is true?

Reply:
Added comment.